electron_app/src/webcontents-handler.js

```diff
@@ -84,14 +84,6 @@ function _CutCopyPasteSelectContextMenus(params) {
     }];
 }
 
-function onSelectedContextMenu(ev, params) {
```

Comment:
this cannot be right, you are removing the Selected context menu, how would highlighting some text need spellchecker?  Surely spellcheck is on inputs rather than highlighted text (which included non-inputs)